packages/hdwallet-native/src/binance.ts

```diff
@@ -0,0 +1,138 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+
+import { addressNListToBIP32, BinanceSignTx, BinanceSignedTx } from "@shapeshiftoss/hdwallet-core";
+
+import BncClient from "bnb-javascript-sdk-nobroadcast";
+import * as bitcoin from "bitcoinjs-lib";
+import { getNetwork } from "./networks";
+import { mnemonicToSeed } from "bip39";
+import { toWords, encode } from "bech32";
+import CryptoJS, { RIPEMD160, SHA256 } from "crypto-js";
+
+function bech32ify(address, prefix) {
+  const words = toWords(address);
+  return encode(prefix, words);
+}
+
+function createBNBAddress(publicKey) {
+  const message = SHA256(CryptoJS.enc.Hex.parse(publicKey.toString(`hex`)));
+  const hash = RIPEMD160(message as any).toString();
+  const address = Buffer.from(hash, `hex`);
+  const bnbAddress = bech32ify(address, `bnb`);
+  return bnbAddress;
+}
+
+export function MixinNativeBinanceWalletInfo<TBase extends core.Constructor>(Base: TBase) {
+  return class MixinNativeBinanceWalletInfo extends Base implements core.BinanceWalletInfo {
+    _supportsBinanceInfo = true;
+
+    async binanceSupportsNetwork(): Promise<boolean> {
+      return true;
+    }
+
+    async binanceSupportsSecureTransfer(): Promise<boolean> {
+      return false;
+    }
+
+    binanceSupportsNativeShapeShift(): boolean {
+      return false;
+    }
+
+    binanceGetAccountPaths(msg: core.BinanceGetAccountPaths): Array<core.BinanceAccountPath> {
+      return [
+        {
+          addressNList: [0x80000000 + 44, 0x80000000 + 117, 0x80000000 + msg.accountIdx, 0, 0],
+          // hardenedPath: [0x80000000 + 44, 0x80000000 + core.slip44ByCoin(msg.coin), 0x80000000 + msg.accountIdx],
+          // relPath: [0, 0],
+          // description: "Native",
+        },
+      ];
+    }
+
+    binanceNextAccountPath(msg: core.BinanceAccountPath): core.BinanceAccountPath {
+      // Only support one account for now (like portis).
+      return undefined;
+    }
+  };
+}
+
+export function MixinNativeBinanceWallet<TBase extends core.Constructor>(Base: TBase) {
+  return class MixinNativeBinanceWallet extends Base {
+    _supportsBinance = true;
+    #seed = "";
+
+    binanceInitializeWallet(seed: string): void {
+      //get
+      this.#seed = seed;
+    }
+
+    bech32ify(address: ArrayLike<number>, prefix: string): string {
+      const words = toWords(address);
+      return encode(prefix, words);
+    }
+
+    createBinanceAddress(publicKey: string) {
+      const message = SHA256(CryptoJS.enc.Hex.parse(publicKey));
+      const hash = RIPEMD160(message as any).toString();
+      const address = Buffer.from(hash, `hex`);
+      const cosmosAddress = this.bech32ify(address, `bnb`);
+      return cosmosAddress;
+    }
+
+    async binanceGetAddress(msg: core.BinanceGetAddress): Promise<string> {
+      const seed = await mnemonicToSeed(this.#seed);
+
+      const network = getNetwork("bitcoin");
```

Comment:
add `binance` to `networks.ts` and just set to use the `bitcoin` network for now.